Add documentation about omitted attributes during IIDM export (#4004)
powsybl/powsybl-core: docs/grid_exchange_formats/iidm/export.md

- JSON: "JIIDM"
- binary: "BIIDM"

> ## Export omitted attributes

Reviewer: Omitted attributes at export

> For performance reasons, from IIDM versions > 1.16, the export performs a reduction of serialized file size by omitting attributes equal to their default value.

Reviewer: ```suggestion
For performance reasons, in IIDM versions > 1.16, the export reduces the size of the serialized file by omitting attributes equal to their default value.
```

Older IIDM versions exports remain unchanged: all attributes continue to be written unconditionally, preserving full backward compatibility.

Here is the full list of attributes now omitted when equal to their default :

| Attribute               | Type                             | Changed to              |
|-------------------------|----------------------------------|-------------------------|
| retained	               | Switch                           | optional default false  |
| r, x, g, b              | RatioTapChangerStep              | optional default 0      |
| r, x, g, b              | PhaseTapChangerStep              | optional default 0      |
| lowTapPosition          | RatioTapChanger, PhaseTapChanger | optional default 0      |
| g1, g2, b1, b2          | Line                             | optional default 0      |
| g, b                    | TwoWindingsTransformer           | optional default 0      |
| g1, g2, g3, b1, b2, b3  | ThreeWindingsTransformer         | optional default 0      |

> ## Exporting with default values

Reviewer: The title of this section is unclear to me, especially with the section just above! At first, it could give the impression that you can, in fact, export with the aforementioned omitted default values.

```suggestion
## Standard export
```
